lib/plugins/package/lib/zipService.js

```diff
@@ -164,14 +168,17 @@ function excludeNodeDevDependencies(servicePath) {
     })
     .then(() => fs.readFileAsync(nodeDevDepFile))
     .then((fileContent) => {
-      const dependencies = _.uniq(_.split(fileContent.toString('utf8'), '\n'));
+      const dependencies = _.filter(
```

Comment:
Could you use `_.compact` here instead of `_.filter`?
https://lodash.com/docs/4.17.4#compact